src/content/en/2022/privacy.md

```diff
@@ -1,22 +1,466 @@
+  )
+}}
+
+By default, any (third-party) script can access the same browser features as the website they're embedded in. In order to limit the features that will be enabled for the website, the website can make use of the [Permissions Policy](https://developer.chrome.com/en/docs/privacy-sandbox/permissions-policy/). Through an HTTP response header the website can indicate which features it wants enabled. For instance, if the `microphone` feature is not included in this list, none of the scripts embedded in the web page can use it. Although the policy is fairly new, we are seeing an adoption of 2.71% on desktop sites and 2.31% on mobile sites.
+The Permissions Policy supersedes the [Feature Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy), which can still be found on 0.69% of desktop sites and 0.52% of mobile sites.
+By default most of the features regulated by the Permissions Policy are disabled in cross-origin iframes, they can be explicitly enabled through the `allow` attribute. We find that 15.18% of desktop sites and 14.32% of mobile sites make use of this feature. For a more detailed analysis on the use of the `allow` attribute on iframes, please refer to [the security chapter](./security).
+
+{{ figure_markup(
+  image="permission-policy-features.png",
+  caption="Most common Permissions Policy feature names.",
+  description="Bar chart showing which Permissions Policy features are most prevalent. The feature `interest-cohort` of the Permissions Policy was present on 1.18% desktop and 0.93% mobile sites respectively, the `geolocation` feature was specificied on 0.80% and 0.58% sites, `microphone` on 0.78% and 0.57%, `camera` on 0.75% and 0.55%, `payment` on 0.57% and 0.42%, `gyroscope` on 0.54% and 0.40%, `magnetometer` on 0.54% and 0.39%, `fullscreen` on 0.47% and 0.34%, `usb` on 0.45% and 0.33%, and finally `accelerometer` was defined on 0.44% desktop and 0.32% mobile sites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRSnqWDZQ07x6NdnIix7JOfg5DqGcoTrrpwaKavCIvNFkIANncyMxahKaTYjLka3rsrcBmrRIURpnPo/pubchart?oid=1803269356&format=interactive",
+  sheets_gid="315730924",
+  sql_file="most_common_permissions_policy_directives.sql"
+  )
+}}
+
+When we look at the directives that are used in the Permissions Policy, we see a similar usage compared to [last year](../2021/privacy), with the exception of the one that's most widely used in 2022, namely `interest-cohort`. This directive can be used to limit the access to the now-defunct FLoC API. Presumably, this can be attributed to the various shortcomings of FLoC (increases fingerprinting surface, reveals potentially sensitive information about users, ...) where website owners took an active step in trying to protect the privacy of their users.
+
+### Referrer Policy
+
+{{ figure_markup(
+  caption="The percentage of desktop sites that sets a document-wide referrer policy.",
+  content="11.64%%",
```

Comment:
Throughout, we should standardize on two significant figures in stats. For example "0.33%" and "12%" but not rounding to "0%" or showing unnecessary precision like "11.64%".
```suggestion
  content="12%",
```

Reply:
Standardised all numbers to two significant figures, except when more decimal places were used in the sentence. Used the following regex to detect such cases: `(\d\d|[1-9])\.\d\d%(?!.*\s0\.\d\d%).*$`